src/components/MyPools/MyPools.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { useAsync } from 'react-use';
+import { makeStyles } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+
+import PoolDisplayCardWide from '../PoolDisplayCard/PoolDisplayCardWide';
+import { useAccountPoolsState } from '../../services/thegraph';
+import { fetchAllPools } from '../../ethereum/pool';
+import { lowercaseAddress } from '../../utils/utils';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    flexDirection: 'column',
+    margin: 'auto',
+    width: '50em',
+    [theme.breakpoints.down('sm')]: {
+      width: 'auto',
+      overflow: 'hidden',
+    },
+  },
+  title: {
+    padding: '1.5em 0',
+    fontWeight: 'bold',
+  },
+  link: {
+    textDecoration: 'none',
+  },
+}));
+
+const MyPools = (props) => {
+  const classes = useStyles();
+  const joinedPoolsAddresses = [];
+  let joinedPools = [];
+  let createdPools = [];
+
+  const poolsMetadata = useAsync(async () => {
+    return fetchAllPools();
+  }, []);
+
+  const accountPools = useAccountPoolsState(props.address);
```

Comment:
this should give all (and only) the joined pools, no?

Reply:
yes

Reply:
Ok I misunderstand it before..
but It's kinda strange you fetch all the pools here, but that looks like the easiest now 👍

Reply:
if you mean by the poolsMetadata, then yes. 
Initially I thought of creating a backend endpoint to fetch specific pools, but It's indeed easier and faster like this.
We would actually need to create 2 endpoints - one for fetching the specific pools (by contract address), and the other for fetching the pools created by user.
I preferred to go with the short solution for now.

Reply:
Agree, I though about this for some time and got to the same conclusions.

Reply:
Anyway you can merge :)